.github/workflows/playground-check.yaml

```diff
@@ -66,3 +66,13 @@ jobs:
         with:
           name: logs.tgz
           path: ./logs.tgz
+
+      - name: Slack notification on failure
+        if: failure()
+        id: slack
+        uses: slackapi/slack-github-action@v1.27.0
+        with:
+          payload: |
+            {"job_link":"${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}"}
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_PLAYGROUND_CI_WEBHOOK_URL }}
```

Comment:
To which channel is it going to report? Also, it seems like it doesn't contain any meaningful message, just a link.

Also, a new line is missing.

Reply:
I've added more information on the context of the fail.

Reply:
Also I've added button to download docker logs stored as artifact.
After testing on #alerts-test channel I will switch that to #alerts-barn (confirmed that with Martin).